tools/lsvast/src/print_segment.cpp

```diff
@@ -35,7 +35,8 @@ print_segment_contents(indentation&, const options&, vast::chunk_ptr chunk) {
   auto writer = vast::format::json::writer{
     std::make_unique<std::stringstream>(), settings};
   for (const auto& slice : *segment)
-    writer.write(slice);
+    if (auto err = writer.write(slice))
+      fmt::print(stderr, "Err during segment slice write {}", err);
```

Comment:
We probably want to `exit(1)` here if printing failed.

I propose changing the error message like this:

```cpp
fmt::print(stderr, "!! failed to write segment slice: {}", err);
```